### DISCUSSION TITLE
Translating Zepp OS

### DISCUSSION THREAD

**Discussion by naskogg** [zepp-health/zeppos-forum#206] [Q&A]
Hello, is it possible to develop translations for the watch and mobile application? I have searched through the discussions but couldn't find any information. Thanks!

**Comment by narky** [comment]
Do you mean translatrion work for native watch and Zepp App on mobile?

**Comment by naskogg** [comment]
Yes, I mean translation work for both of them, but the watch is a priority.

**Comment by MyoungXUE** [comment]
Hello, you can sort out the flow of this translation program, some technical implementation details, and we will help you evaluate the feasibility of the solution.
